web/client/utils/mapinfo/wfs.js

```diff
@@ -75,7 +90,14 @@ const getIdentifyGeometry = point => {
 export default {
     buildRequest,
     getIdentifyFlow: (layer, baseURL, defaultParams) => {
-        const { point, ...baseParams} = defaultParams;
+        const { point, features = [], ...baseParams} = defaultParams;
+        if (features.length) {
```

Comment:
Here we should prevent to make request even if the features array is empty because it could be a valid response.
Please let's remove the empty array fallback and check only if features is provided in the params
```suggestion
        const { point, features, ...baseParams} = defaultParams;
        if (features) {
```